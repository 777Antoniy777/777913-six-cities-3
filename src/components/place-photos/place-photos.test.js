import React from 'react';
import renderer from "react-test-renderer";
import PlacePhotos from './place-photos';

// set mocha data
const photos = [
  `img/image1`,
  `img/image2`,
  `img/image3`,
];

let i = 0;

it(`render PlacePhotos`, () => {
  const tree = renderer.create(
      <PlacePhotos
        key={++i}
        photos={photos}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
