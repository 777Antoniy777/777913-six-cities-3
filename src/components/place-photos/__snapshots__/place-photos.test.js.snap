// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PlacePhotos 1`] = `
<div
  className="property__gallery"
>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/image1"
    />
  </div>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/image2"
    />
  </div>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="img/image3"
    />
  </div>
</div>
`;
